docs/nodes/maintain/avalanchego-config-flags.md

```diff
@@ -1308,6 +1308,10 @@ GiB).
 
 If true, runs the node as a [plugin.](https://github.com/hashicorp/go-plugin) Defaults to `false`.
```

Comment:
Can you remove this `--plugin-mode-enabled` flag as it's not used anymore?

Reply:
fixed 4abab397e720b274ee80650ca44f211076a68a67